### PR TITLE
LF-2376/notify user when task reassigned to someone else

### PR DESCRIPTION
**Pull request by seinnlee**
Ticket [LF-2376](https://lite-farm.atlassian.net/browse/LF-2376)

**Description**
We want to notify a user when their task has been reassigned by assigner (i.e. farm owner or manager) to someone else.

**To Test:**
1. You will need at least one worker at your farm that you have access to.
2. Create a task and assign it to the worker from step 1.
3. Reassign the same task to someone else.
4. Log in to the account of the worker you initially assigned the task to from step 1.
5. You will see two notifications (initial assignment notification and reassignment notification)

I use the currently authenticated user's id as the assigner from request headers.

### CODE REVIEW

**Review by Cali0707 — APPROVED** [review]
I tested the functionality as described and it works as expected - both notifications showed up. This looks good to me.

**Review by smattingly — CHANGES_REQUESTED** [review]
Good work. I'd like to have a back-end Jest test for this. @seinnlee let me know if you need help with that. Also, if writing the test risks sprint tickets going unfinished, contact me and we can delay it by creating a tech debt ticket.

**Comment by seinnlee** [comment]
> Good work. I'd like to have a back-end Jest test for this. @seinnlee let me know if you need help with that. Also, if writing the test risks sprint tickets going unfinished, contact me and we can delay it by creating a tech debt ticket.

@smattingly I'll need some help with writing tests. I've looked into it and it looks like the test will have to call both assignment endpoint and notification endpoint to test this functionality.

File: packages/api/tests/taskNotification.test.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
+/*
+ *  Copyright 2019-2022 LiteFarm.org
```
Comment (Cali0707) [inline packages/api/tests/taskNotification.test.js:2]:
```suggestion
 *  Copyright 2019, 2020, 2021, 2022 LiteFarm.org
```

**Review by Cali0707 — APPROVED** [review]
One small change in a copyright comment should be made, bu otherwise this looks good to me!

**Review by fsayyid — APPROVED** [review]
Everything is working on my end and all tests are passing. @smattingly passing this off to you.

**Review by smattingly — APPROVED** [review]
Very good!

**Pull request merged**